python/friesian/dev/test/run-example-tests-tf2.6.0.sh

```diff
@@ -112,6 +112,26 @@ python ../../example/deep_fm/deepFM_train.py \
 now=$(date "+%s")
 time5=$((now - start))
 
+echo "#6 start example test for multi task train"
```

Comment:
I think we need to put these two tests in a single script, use the processed data to run train